### DISCUSSION TITLE
A different way to style rows

### DISCUSSION THREAD

**Discussion by nardove** [Comcast/react-data-grid#3086] [Q&A] [answered]
Is there a different way to style rows not using @linaria/core ?

**Comment by nstepien** [comment, marked as answer]
You can set the `className` or `style` prop on the root `DataGrid` component like a normal `div`, or you can use the `rowClass` prop as a function that returns a class name.
How you define your CSS styles and class names is up to you.

**Reply by nardove** [reply]
@nstepien I see, this is what I'm doing now

`rowClass={(row) => (row.type === 'DETAIL' ? 'detail_row' : undefined)}`

Where `detail_row` is a class name in the component scss imported file, but the styles don't seem to take effect, am I missing something else?

**Reply by nardove** [reply]
Forget it, what I did does work, thank you!
